lib/api/BackbeatAPI.js

```diff
@@ -736,6 +736,9 @@ class BackbeatAPI {
         }
         try {
             reqBody = JSON.parse(body);
+            if (reqBody.cancel) {
```

Comment:
If a user set this to 'false', wouldn't this skip?
In the documentation it says:
"Providing a POST request body object with a `cancel` key with any value will remove any scheduled resumes for the given destination replication endpoint."

Reply:
ah good catch! I think I'm going to change this so rather than any value, I expect user to send `true`. Seems sort of weird to just accept any value from a ux perspective